tests/src/main/java/net/neoforged/neoforge/debug/client/ClientEventTests.java

```diff
@@ -126,4 +140,47 @@ static void renderPlayerEvent(final DynamicTest test) {
             test.requestConfirmation(Minecraft.getInstance().player, Component.literal("Is an iron block rendered above you in third-person?"));
         });
     }
+
+    @TestHolder(description = { "Test render state modifier system and registration event" })
+    static void updateRenderState(final DynamicTest test) {
+        var rotationKey = new ContextKey<Float>(ResourceLocation.fromNamespaceAndPath(test.createModId(), "rotation"));
+        var numRenderAttachmentKey = new ContextKey<Integer>(ResourceLocation.fromNamespaceAndPath(test.createModId(), "times_to_render"));
+        var testAttachment = test.registrationHelper().attachments().registerSimpleAttachment("test", () -> 3);
+        test.eventListeners().mod().addListener((RegisterRenderStateModifiersEvent event) -> {
```

Comment:
```suggestion
        test.framework().modEventBus().addListener((RegisterRenderStateModifiersEvent event) -> {
```
This must use `test.framework().modEventBus()` instead of `test.eventListeners().mod()`, otherwise the listener won't be invoked.

Reply:
Huh. Changed it, a little annoying that there are two similar ways that seem to be correct, but one is only active when the test is enabled...